src/App.tsx

```diff
@@ -21,6 +37,11 @@ export const App: React.FC = () => {
             data-cy="perPageSelector"
             id="perPageSelector"
             className="form-control"
+            defaultValue={5}
+            onChange={(event) => {
+              setPerPage(+event.target.value);
+              setCurrentPage(1);
```

Comment:
it is bad practise to make anonymous arrow functions in jsx.
make handler in component and pass reference here

check other places